docs/howto_wallet_polkadot_vault.md

```diff
@@ -1,11 +1,13 @@
 ---
-id: howto_wallet_parity_signer
-title: Use Parity Signer
+id: howto_wallet_polkadot_vault
+title: Polkadot Vault
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
```

Comment:
this needs to be updated for Polkadot Vault (i just changed the title so we can link in the newsletter)